### PR TITLE
Fix profiles in the weekly update playbooks

### PR DESCRIPTION
**Pull request by bitrisekristof**

### CODE REVIEW

File: roles/patch_bashrc/tasks/main.yml [hunk 1]
```diff
@@ -2,19 +2,17 @@
...
-  become: yes
-  blockinfile:
-    insertbefore: "keep a newline"
-    path: /Users/{{ param_user }}/.bashrc
-    block: "{{ lookup('template', 'patch.j2') }}"
-    marker: "# {mark} ANSIBLE MANAGED BLOCK - Android Additions"
+- name: Creates profile.d directory
+  file:
+    path: $HOME/profile.d
+    state: directory
```
Comment (ghaabor) [inline roles/patch_bashrc/tasks/main.yml]:
```suggestion
    state: directory
    owner: vagrant
    group: vagrant
    mode: 0660
```

File: roles/patch_bashrc/tasks/main.yml [hunk 2]
```diff
@@ -2,19 +2,17 @@
...
-    insertbefore: "keep a newline"
-    path: /Users/{{ param_user }}/.bashrc
-    line: ""
+- name: Create patch.sh to profile.d
+  template:
+    src: patch.j2
+    dest: $HOME/profile.d/patch.sh
+    owner: vagrant
+    group: vagrant
+    mode: 0660
```
Comment (ghaabor) [inline roles/patch_bashrc/tasks/main.yml]:
```suggestion
    mode: 0770
```
660 is not executable

File: roles/jenv/tasks/main.yml [hunk 3]
```diff
@@ -7,20 +7,18 @@
...
+    path: $HOME/profile.d
+    state: directory
+
+- name: Create patch.sh to profile.d
+  template:
+    src: patch.j2
+    dest: $HOME/profile.d/patch.sh
+    owner: vagrant
+    group: vagrant
+    mode: 0660
```
Comment (ghaabor) [inline roles/jenv/tasks/main.yml]:
```suggestion
    mode: 0770
```

File: roles/jenv/tasks/main.yml [hunk 4]
```diff
@@ -7,20 +7,18 @@
...
-- name: Final newline
-  become: yes
-  lineinfile:
-    insertbefore: "keep a newline"
-    path: /Users/{{ param_user }}/.bashrc
-    line: ""
+- name: Creates profile.d directory
+  file:
+    path: $HOME/profile.d
+    state: directory
```
Comment (ghaabor) [inline roles/jenv/tasks/main.yml]:
```suggestion
    state: directory
    owner: vagrant
    group: vagrant
    mode: 0660
```

File: roles/flutter/tasks/main.yml [hunk 5]
```diff
@@ -9,20 +9,18 @@
...
-    insertbefore: "keep a newline"
-    path: /Users/{{ param_user }}/.bashrc
-    line: ""
+- name: Create patch.sh to profile.d
+  template:
+    src: patch.j2
+    dest: $HOME/profile.d/patch.sh
+    owner: vagrant
+    group: vagrant
+    mode: 0660
```
Comment (ghaabor) [inline roles/flutter/tasks/main.yml]:
```suggestion
    mode: 0770
```

File: roles/flutter/tasks/main.yml [hunk 6]
```diff
@@ -9,20 +9,18 @@
...
-  become: yes
-  blockinfile:
-    insertbefore: "keep a newline"
-    path: /Users/{{ param_user }}/.bashrc
-    block: "{{ lookup('template', 'patch.j2') }}"
-    marker: "# {mark} ANSIBLE MANAGED BLOCK - Flutter settings"
+- name: Creates profile.d directory
+  file:
+    path: $HOME/profile.d
+    state: directory
```
Comment (ghaabor) [inline roles/flutter/tasks/main.yml]:
```suggestion
    state: directory
    owner: vagrant
    group: vagrant
    mode: 0660
```

File: profiles/bashrc [hunk 7]
```diff
@@ -14,6 +14,10 @@ export PATH="$PATH:/usr/local/opt/go/libexec/bin"
 export GOPATH="$HOME/go"
 export PATH="$PATH:$GOPATH/bin"
 
+for PROFILE_SCRIPT in $( ls $HOME/profile.d/*.sh ); do
```
Comment (ghaabor) [inline profiles/bashrc]: Please move the for cycle to the end of the file

File: roles/patch_bashrc/tasks/main.yml [hunk 8]
```diff
@@ -2,19 +2,17 @@
...
-- name: Final newline
-  become: yes
-  lineinfile:
-    insertbefore: "keep a newline"
-    path: /Users/{{ param_user }}/.bashrc
-    line: ""
+- name: Create patch.sh to profile.d
+  template:
+    src: patch.j2
+    dest: $HOME/profile.d/patch.sh
```
Comment (ghaabor) [inline roles/patch_bashrc/tasks/main.yml]: Rename it from patch.sh to <role-name>.sh, or else the scripts will overwrite each other.

File: roles/jenv/tasks/main.yml [hunk 9]
```diff
@@ -7,20 +7,18 @@
...
+
+- name: Create patch.sh to profile.d
+  template:
+    src: patch.j2
+    dest: $HOME/profile.d/patch.sh
```
Comment (ghaabor) [inline roles/jenv/tasks/main.yml]: Rename it from patch.sh to <role-name>.sh, or else the scripts will overwrite each other.

File: roles/flutter/tasks/main.yml [hunk 10]
```diff
@@ -9,20 +9,18 @@
...
-- name: Flutter final newline in bashrc
-  become: yes
-  lineinfile:
-    insertbefore: "keep a newline"
-    path: /Users/{{ param_user }}/.bashrc
-    line: ""
+- name: Create patch.sh to profile.d
+  template:
+    src: patch.j2
+    dest: $HOME/profile.d/patch.sh
```
Comment (ghaabor) [inline roles/flutter/tasks/main.yml]: Rename it from patch.sh to <role-name>.sh, or else the scripts will overwrite each other.

File: roles/flutter/tasks/main.yml (hunk 6 above)
Comment (bitrisekristof) [reply roles/flutter/tasks/main.yml]: done

**Pull request merged**